### ISSUE TITLE
Who emoted at me?

### ISSUE THREAD

**Issue by clytemnestra** [AjisaiMirai/plugins#4]
Hello, what happened to that plugin? It was really useful 😢 

I figured out it was crashing my game after yesterday's patch and following here I don't see the repo anymore.

**Comment by AjisaiMirai** [comment]
Hello!

Who Emoted At Me should still be a plugin on this repo. It’s still present in “repo.json” and in the releases so it should be downloadable. 

About the crashing issue; I’m aware and will look at it when I’m back behind my own machine. The hook is rather messy so I’ll firstly look into cleaning that up and structuring it.
 
My guess to the crashing is probably caused by my quick implementation of a macro press.
Basically the idea originated with a friend to send an /em whenever receiving a pet emote. Whenever this is registered and you have an in-game macro made and bound to “Page_up” it would trigger. 
Shit implementation but it works and I wasn’t able to figure out how to directly call the “em” function, lol. 

I’ll make sure it’s functional again somewhat this week

**Comment by AjisaiMirai** [comment]
Updated to v26
